pkgs/development/python-modules/shapely/default.nix

```diff
@@ -44,12 +44,6 @@ buildPythonPackage rec {
       libgeos_c = GEOS_LIBRARY_PATH;
       libc = lib.optionalString (!stdenv.isDarwin) "${stdenv.cc.libc}/lib/libc${stdenv.hostPlatform.extensions.sharedLibrary}.6";
     })
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
done